src/main/java/network/radicle/jetbrains/radiclejetbrainsplugin/patches/PatchComponentFactory.java

```diff
@@ -71,15 +71,15 @@ private void calculatePatchChanges(GitRepository repo, GitLocalBranch localBranc
     }
 
     private void calculatePatchChanges(RadPatch patch) {
-        List<Change> changes = new ArrayList<>();
-        for (var rev : patch.revisions) {
-            final var diff =  GitChangeUtils.getDiff(patch.repo, rev.base(), rev.oid(), true);
-            if (diff != null && !diff.isEmpty()) {
-                changes.addAll(diff);
-            }
+        if (patch.revisions.isEmpty()) {
+            return;
         }
-        changes = changes.stream().distinct().collect(Collectors.toList());
-        patchChanges.setValue(changes);
+        var latestRevision = patch.revisions.get(patch.revisions.size() - 1);
```

Comment:
this is straight forward, but what is expected to happen when the last revision is not the last one chronologically as well?

Reply:
@Stelios123 I just added another comment there, because there was another relevant discussion.